samples/csharp_dotnetcore/50.teams-messaging-extensions-search/Bots/TeamsMessagingExtensionsSearchBot.cs

```diff
@@ -99,5 +136,153 @@ private async Task<IEnumerable<(string, string, string, string, string)>> FindPa
             var obj = JObject.Parse(await (new HttpClient()).GetStringAsync($"https://azuresearch-usnc.nuget.org/query?q=id:{text}&prerelease=true"));
             return obj["data"].Select(item => (item["id"].ToString(), item["version"].ToString(), item["description"].ToString(), item["projectUrl"]?.ToString(), item["iconUrl"]?.ToString()));
         }
+
+
+        public MessagingExtensionResponse GetAdaptiveCard()
+        {
+
+            string filepath = "./RestaurantCard.json";
+
+            //string filepath = "./WeatherCard.json";
+
+            var previewcard = new ThumbnailCard
+            {
+                Title = "Adaptive Card",
+                Text = "Please select to get Adaptive card"
+            };
+
+            var adaptiveList = FetchAdaptive(filepath);
+            var attachment = new MessagingExtensionAttachment
+            {
+                ContentType = AdaptiveCards.AdaptiveCard.ContentType,
+                Content = adaptiveList.Content,
+                Preview = previewcard.ToAttachment()
+            };
+
+            // return attachment;
+
+            return new MessagingExtensionResponse
+            {
+                ComposeExtension = new MessagingExtensionResult
+                {
+                    Type = "result",
+                    AttachmentLayout = "list",
+                    Attachments = new List<MessagingExtensionAttachment> { attachment }
+                }
+            };
+        }
+
+
+        public MessagingExtensionResponse GetConnectorCard()
+        {
+          
+            string filepath = "./connectorCard.json";
+
+            var json = File.ReadAllText(filepath);
+
+            var cardJson = JsonConvert.DeserializeObject<ConnectorJsonSerializer>(json);
+
+            cardJson.sections[0].activityImage = _configuration+"/imgConnector.jpg";
+
+            var ConnectorCardJson = JsonConvert.SerializeObject(cardJson);
```

Comment:
Please remove the ConnectorJsonSerializer classes, and just use a placeholder string used for the activityImage?